src/containers/Info/Contact.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+
+import { H2, HrefHeader } from '../../components';
+
+import { Contact } from '../../common';
+
+import { Contact as ContactWrapper, ContactHeader, ContactList } from './styled';
+
+export interface LicenseProps {
+  contact: Contact;
+}
+
+class LicenseComponent extends Component<LicenseProps> {
+  public render() {
+    const { contact: { name, url, email } } = this.props;
+
+    return (
+      <ContactWrapper>
+        <ContactHeader>
+          <H2>Contact</H2>
+        </ContactHeader>
+        <ContactList>
+          {name ? <li><strong>Name</strong>: {name}</li> : null}
```

Comment:
Please replace ternary operator with guard whenever you return null when a condition is not met
```
{name && <li><strong>Name</strong>: {name}</li>}
```